### ISSUE TITLE
Revise hover styling for inline body links

### ISSUE THREAD

**Issue by jrbarnes9** [ONSdigital/design-system#1261]
I think we should consider revising the hover styling for inline body links.  We currently remove the underline (to meet WCAG 'don't rely on colour alone' guidance) but, some users have reported the links 'disappearing' during Census UR.  This is due to both the underline being removed, and the link hover colour being black in census theme.

Proposing to keep the underline on hover and increase thickness to 2px using `text-decoration-thickness` as an enhancement

https://user-images.githubusercontent.com/43346934/102881064-36a24080-4444-11eb-85ed-cb37279afad5.mov

_Originally posted by @jrbarnes9 in https://github.com/ONSdigital/design-system/discussions/233#discussioncomment-234233_
